### DISCUSSION TITLE
PCL2的JavaWrapper.jar有什么用

### DISCUSSION THREAD

**Discussion by mc-124** [Meloong-Git/PCL#2945] [提问] [answered]
如题

**Comment by SqrtTan** [comment]
JAVA包装器是个什么东西，打包jar的？

**Reply by mc-124** [reply]
动动脑子，这有可能吗

**Comment by burningtnt** [comment]
JavaWrapper 最主要的作用是处理 LWJGL 的 Native 库

**Comment by hejiehao** [comment]
似乎是用来安装 Modloader 的

**Reply by mc-124** [reply]
不应该，forge有forge-installer，fabric、quilt、optifine那些都带自动安装

**Comment by LTCatt** [comment, marked as answer]
https://github.com/00ll00/java_launch_wrapper
